code/datums/traits/negative.dm

```diff
@@ -579,3 +579,61 @@
 	lose_text = "<span class='notice'>Your mind finally feels calm.</span>"
 	medical_record_text = "Patient's mind is in a vulnerable state, and cannot recover from traumatic events."
 	hardcore_value = 9
+
+/datum/quirk/allergic
+	name = "Extreme Medicine Allergy"
+	desc = "Ever since you were a kid, you've been allergic to certain chemicals..."
+	value = -2
+	gain_text = "<span class='danger'>You feel your immune system shift.</span>"
+	lose_text = "<span class='notice'>You feel your immune system phase back into perfect shape.</span>"
+	medical_record_text = "Patient's immune system responds violently to certain chemicals."
+	hardcore_value = 3
+	var/list/allergies = list()
+	var/list/blacklist = list(/datum/reagent/medicine/c2,/datum/reagent/medicine/epinephrine,/datum/reagent/medicine/adminordrazine,/datum/reagent/medicine/omnizine/godblood,/datum/reagent/medicine/cordiolis_hepatico,/datum/reagent/medicine/synaphydramine,/datum/reagent/medicine/diphenhydramine)
```

Comment:
Shouldn't this be a global or static list?

Reply:
why would it be global???? but yeah it could be a static list

Reply:
The reason for global I was thinking of:

I might make a PR after this gets merged, of a traitor item that adds allergies to someone.  It being global might matter there.

But given that it's for a future PR, it doesn't matter, I can deal with it then.